src/App.tsx

```diff
@@ -1,61 +1,116 @@
 import './App.scss';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
 
 export const App = () => {
+  const [title, setTitle] = useState<string>('');
+  const [tasks, setTasks] = useState<Todo[]>(todosFromServer);
```

Comment:
your `Todo` type should have user, so you need to change initial value first and then assign modified to tasks
the same as you made here, but add null if user is not found 
```
const todos = () => tasks.map(todo => ({
    ...todo,
    user: getUser(todo.userId) (|| null to add) ,
  }));
  ```

Reply:
+1 Currently `todosFromServer` does not have user information. Please assign it first, then initialize the state with a default value.